airflow/config_templates/default_airflow.cfg

```diff
@@ -207,6 +207,9 @@ dag_discovery_safe_mode = True
 # The number of retries each task is going to have by default. Can be overridden at dag or task level.
 default_task_retries = 0
 
+# The weighting method used for the effective total priority weight of the task
+default_task_weight_rule = downstream
```

Comment:
We should probably validate that this option is valid (i.e. it's in the enum) in airflow.configuration.Configration.validate()

Reply:
is https://github.com/apache/airflow/pull/18627/commits/da521bd820ead09cfd6fa1d1b8f1c73f16a66c78 what you had in mind?

Reply:
Yup, thanks!

Reply:
Oh. Though it turns out WeightRule isn't actually an Enum. Go figure.

Reply:
It was supposed to be converted to Enum in https://github.com/apache/airflow/pull/5302